sdk/template-dpg/Azure.ServiceTemplate.Template/tests/TemplateClientTest.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.IO;
+using System.Net.Http;
+using System.Text.Json;
+using System.Threading.Tasks;
+using Azure.Core.Pipeline;
+using Azure.Core.TestFramework;
+using NUnit.Framework;
+
+namespace Azure.ServiceTemplate.Template.Tests
+{
+    public class TemplateClientTest: RecordedTestBase<TemplateClientTestEnvironment>
+    {
+        public TemplateClientTest(bool isAsync) : base(isAsync)
+        {
+        }
+
+        /* Greate the client and instrument it as following. */
```

Comment:
Why are some things commented out and some things not?

Reply:
The commented out code is an example to show how to create client, or write test. Those code have some incorrect service related information such as client type, and cannot pass build, so commented out